.ci-operator.yaml

```diff
@@ -0,0 +1,4 @@
+build_root_image:
+  name: release
+  namespace: openshift
+  tag: rhel-8-release-golang-1.20-openshift-4.16
```

Comment:
Can you make this the same golang-1.21 version as the image in Dockerfile.openshift?